addons/web_editor/static/lib/summernote/src/js/module/Editor.js

```diff
@@ -618,6 +618,9 @@ define([
       var isNewWindow = linkInfo.isNewWindow;
       var rng = linkInfo.range || this.createRange($editable);
       var isTextChanged = rng.toString() !== linkText;
+      // Hack : This method was updated to create buttons as well (using the same logic as anchor nodes).
+      const nodeName = linkInfo.isButton ? 'BUTTON' : 'A';
```

Comment:
@xO-Tx In fact, could you add a test tour for this? Or maybe extend a test in website_form to edit the submit button, check that no URL field is suggested and change its style. Here is an example of test tour we recently merged:https://github.com/odoo/odoo/pull/65551/commits/06c576f776223bc6c1aabdf017600c211461b5e2#diff-d0d175bb216da862c91b1e592a0419875b0d95ccd633f7b58d44d2f444270b4fR11 but I don't think you need to create one from scratch, just extend a website_form tour.